src/tests/ftest/util/daos_core_base.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, *args, **kwargs):
 
         test_timeout = self.params.get("test_timeout", self.TEST_PATH)
         if test_timeout:
+            self.log.info("Test timeout updated to %s", test_timeout)
```

Comment:
Please review https://jira.hpdd.intel.com/browse/DAOS-6184?focusedCommentId=254614&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-254614 and let me know your thoughts.  I think we should still be able to implement what we discussed in apricot Test.